src/ar.android.ts

```diff
@@ -338,10 +386,26 @@ export class AR extends ARBase {
                 } else if (state === com.google.ar.core.TrackingState.TRACKING) {
                   // Create a new anchor for newly found images.
                   if (augmentedImages.indexOf(augmentedImage.getName()) === -1) {
-                    const node = new com.google.ar.sceneform.AnchorNode(augmentedImage.createAnchor(augmentedImage.getCenterPose()));
+                    const anchor = new com.google.ar.sceneform.AnchorNode(augmentedImage.createAnchor(augmentedImage.getCenterPose()));
 
                     augmentedImages.push(augmentedImage.getName());
-                    scene.addChild(node);
+                    scene.addChild(anchor);
+                  
+                    const planeNode=new com.google.ar.sceneform.Node();
+                    anchor.addChild(planeNode);
+                    
+                    planeNode.setLocalRotation(new (<any>com.google.ar.sceneform).math.Quaternion(
+                        new (<any>com.google.ar.sceneform).math.Vector3(
+                            -90, 
+                            0,
+                            0
+                        )
+                    ));
+
+
+                    //TODO calculate actual proper scale factor. for ios, tracking images have a defined width - I 
+                    // believe the scale factor is measured-width/defined-width in ios...
+                    planeNode.setLocalScale(new (<any>com.google.ar.sceneform).math.Vector3(0.025, 0.025, 0.025));
```

Comment:
I've added this scale factor for now, but it needs to be calculated as per the comment...
for the pokemon card Bulbasour, the IOS defined width is 4 meters but it appears to be about 0.1m on my computer
so thats where 0.025 comes from

Reply:
Updated this to calculate scale factor but using 4m as defined width of image